src/DevTools/setupTestWrapper.tsx

```diff
@@ -56,13 +66,23 @@ type SetupTestWrapper<T extends OperationType> = {
 
   // If wanting to manually provide fixture data, can pass in an object that
   // maps to the type and field resolvers.
-  renderWithRelay({
+  const { expectAndHandleOperation } = renderWithRelay({
     Show: () => ({
       description: "Hello world!"
     })
   })
 
-	expect(screen.queryByText("Hello world")).toBeInTheDocument()
+  // Manually resolve pending mutations:
+  expect(screen.queryByText("Hello world")).toBeInTheDocument()
+  userEvent.click(screen.getByText("Click me"))
+
+  await expectAndHandleOperation({
+    name: "clickIncrementMutation",
```

Comment:
Why do we need the name? The name should be derived from the operation itself (does relay test utils return this metadata?) and returned to be asserted in actual test code

Reply:
yes, it's available inside, though i re-wrote it now to be optional and only match if provided. The helper is already an expectation (if no operation happens it will throw an error), and I though it semantically useful to let the developer make their intent and other expectations explicit:
- i expect there to be an operation
- i expect it to be this operation
- i expect it to send this input
- it should resolve/reject with these values i care about

Reply:
I'm kinda super against this(wanted to emphasize really hard here  😂), I had a nightmare refactoring these codes in the order app tests to unblock the relay upgrade mostly because of a lot of these abstractions involving this approach expecting a specific name from a mutation.
I will come with a decent explanation why tomorrow